CONTRIBUTING.md

```diff
@@ -2,6 +2,10 @@
 
 This project welcomes contributions, suggestions, and feedback. All contributions, suggestions, and feedback you submitted are accepted under the [Project's license](./LICENSE.md). You represent that if you do not own copyright in the code that you have the authority to submit it under the [Project's license](./LICENSE.md). All feedback, suggestions, or contributions are not confidential.
 
+## Tenets
+
```

Comment:
Instead of adding new section, move this link the list of procedures contributors should follow:

```
* [Project Tenants][./contributions/project-tenets.md]
* [Code Contributions](contributions/code-contributions.md)
* [Opening and Assessing Issues](contributions/opening-and-assessing-issues.md)	* [Opening and Assessing Issues](contributions/opening-and-assessing-issues.md)
* [Approving Pull Requests](contributions/merging-pull-requests.md)	* [Approving Pull Requests](contributions/merging-pull-requests.md)
* [Versioning and Releases](contributions/versioning-and-releases.md)	* [Versioning and Releases](contributions/versioning-and-releases.md)
```

Reply:
sounds good